doc/assessment.md

```diff
@@ -143,6 +143,8 @@ PrairieLearn is designed to give students immediate feedback on their work. Howe
 
 To disable real-time grading for an assessment, add `"allowRealTimeGrading": false` to the assessment's `infoAssessment.json` file. This will hide the "Save & Grade" button on student question pages; only the "Save" button will be available. The "Grade saved answers" button on the assessment overview will also be hidden. Note that real-time grading can only be disabled for `Exam` assessments, as immediate feedback is a core part of the `Homework` experience.
 
+An assessment without real-time grading will not show any score information during the exam, but after a [time limit](accessControl.md#time-limits) runs out the assessment with auto-grade and show students exactly which questions they got correct/incorrect. To prevent this, set the [`showClosedAssessment` access rule restriction](accessControl.md#showinghiding-closed-assessments).
```

Comment:
```suggestion
An assessment without real-time grading will not show any score information during the exam, but after the [time limit](accessControl.md#time-limits) runs out the assessment will auto-grade and show students exactly which questions they got correct/incorrect. To prevent this, set the [`showClosedAssessment` access rule restriction](accessControl.md#showinghiding-closed-assessments).
```

Reply:
I used `a time limit` because there might not be one. Let me reword a bit more.